packages/product/src/migrations/InitialSetup20240312114652.ts

```diff
@@ -0,0 +1,184 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class InitialSetup20240312114652 extends Migration {
```

Comment:
The migration does the bare minimum to make it compatible in the tests with the existing tables, there is more work left to be done here